hoomd_benchmarks/common.py

```diff
@@ -109,20 +109,23 @@ def execute(self):
         print_verbose_messages = (self.verbose
                                   and self.device.communicator.rank == 0)
 
+        # Ensure that all ops are attached (needed for is_tuning_complete).
+        self.sim.run(0)
```

Comment:
```suggestion
        self.run(0)
```
Nitpicky, but might as well just call `self.run()` here.